src/components/interface/mem_mgr/stubs/s_stub.S

```diff
@@ -10,8 +10,15 @@
 
 .text	
 
-cos_asm_server_stub_spdid(mman_get_page)
+//FIXME: the kernel ensured _spdid is not passed
+
+//cos_asm_server_stub_spdid(mman_get_page)
+//cos_asm_server_stub_spdid(mman_revoke_page)
+//cos_asm_server_stub_spdid(__mman_alias_page)
+
+cos_asm_server_stub(mman_get_page)
```

Comment:
Shouldn't this be a new interface?  Isn't this breaking every other use of the memory manager?